<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>journalView</class>
 <widget class="QWidget" name="journalView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>960</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>SolaimanLipi</family>
    <pointsize>14</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>দৈনিক হিসাব</string>
  </property>
  <widget class="QLabel" name="label_grace">
   <property name="geometry">
    <rect>
     <x>436</x>
     <y>0</y>
     <width>91</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>আল্লাহ ভরসা</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_c_in">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>220</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string>নগদ জমা</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label_name">
         <property name="text">
          <string>নাম</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_address">
         <property name="toolTip">
          <string/>
         </property>
         <property name="text">
          <string>ঠিকানা</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_amount">
       <property name="toolTip">
        <string>মন্তব্য</string>
       </property>
       <property name="text">
        <string>০০০০৳</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_c_out">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>50</y>
     <width>220</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string>নগদ খরচ</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>201</width>
      <height>52</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_name_3">
         <property name="text">
          <string>নাম</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_address_3">
         <property name="text">
          <string>ঠিকানা</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="label_amount_3">
       <property name="toolTip">
        <string>মন্তব্য</string>
       </property>
       <property name="text">
        <string>০০০০৳</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignBottom|Qt::AlignRight|Qt::AlignTrailing</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_inventory_in">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>50</y>
     <width>220</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string>Inventory In</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox_inventory_out">
   <property name="geometry">
    <rect>
     <x>730</x>
     <y>50</y>
     <width>220</width>
     <height>500</height>
    </rect>
   </property>
   <property name="title">
    <string>Inventory Out</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_date_day">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>251</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>SolaimanLipi</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Date: 01/01/2020 Day: Saturday</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="pbViewJournalPrevious">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pbViewJournalNext">
   <property name="geometry">
    <rect>
     <x>640</x>
     <y>10</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_newTrx_in">
   <property name="geometry">
    <rect>
     <x>114</x>
     <y>30</y>
     <width>111</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>জমা</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pb_newTrx_out">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>101</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>খরচ</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
